Python/sumofnaturalnos.py

```diff
@@ -0,0 +1,13 @@
+def sumofnaturalno(num):
+    if num < 0:
+       print("Enter a positive number")
+    else:
+       sum = 0
+       # use while loop to iterate until zero
+       while(num > 0):
+           sum += num
+           num -= 1
+       print("The sum is", sum)
+
```

Comment:
You can add another method as a comment. Simple maths formula of finding the sum of first n natural numbers as n(n+1)/2 in py.

```py
# Another Method

def findSum(num) :
    return num * (num + 1) / 2
n = int(input())
print findSum(n)
```

Reply:
You can wish to add this method as comment if you want to give more insight to beginners.

Reply:
Please review or you can also cancel the changes made by me if they are not necessary for the code.

Reply:
You can create an issue and work on it, I'll merge them:)

Reply:
Okay. will do.